avalon/photoshop/lib.py

```diff
@@ -177,7 +177,7 @@ def _recurse_layers(layers):
     for layer in layers:
         result[layer.id] = layer
         if layer.LayerType == com_objects.constants().psLayerSet:
-            result.update(_recurse_layers([*layer.Layers]))
+            result.update(_recurse_layers([x for x in layer.Layers]))
```

Comment:
Isn't `[x for x in layer.Layers]` similar to `list(layer.Layers)` @tokejepsen?

Reply:
Probably is. Do you want it changed?

Reply:
Not necessarily but I believe that's what is officially PEP08 recommended, but couldn't find it just now.